agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -153,7 +156,17 @@ public void customize(AutoConfigurationCustomizer autoConfiguration) {
 
     // interval longer than 15 minutes is not allowed since we use this data for usage telemetry
     long intervalSeconds = Math.min(config.heartbeat.intervalSeconds, MINUTES.toSeconds(15));
-    HeartBeatProvider.start(intervalSeconds, telemetryClient);
+    Consumer<List<TelemetryItem>> telemetryItemsConsumer =
+        telemetryItems -> {
+          for (TelemetryItem telemetryItem : telemetryItems) {
+            TelemetryObservers.INSTANCE
+                .getObservers()
+                .forEach(consumer -> consumer.accept(telemetryItem));
```

Comment:
this accept does the populateDefaults @trask? i might need to remove telemetryinitializer in HeartbeatExporter.

Reply:
no it doesn't. please ignore this. it was passed to MetricDataMapper. we need to keep it